playwright/mockData/orgs/KPD/journeys/JourneyInstances.ts

```diff
@@ -0,0 +1,110 @@
+import Chance from 'chance';
+import dayjs from 'dayjs';
+import { uniqBy } from 'lodash';
+
+import mockJourneyInstance from 'utils/testing/mocks/mockJourneyInstance';
```

Comment:
Im surprised that this works. Was this fixed by adding `downLevelIteration` to the tsconfig? Either way, amazing. Super happy to have this working because those long relative imports were a pain

Reply:
You know I had not actually noticed - I refactored it in there from within /src somewhere but didn't edit the imports 😂